plugin/src/test/scala/io/github/retronym/classpathshrinker/TestUtil.scala

```diff
@@ -62,22 +63,41 @@ object TestUtil {
     reporter.infos
       .map { info =>
         if (info.pos == NoPosition) info.msg
-        else s"""[${info.pos.source}]:${info.pos.line}: ${info.msg}"""
+        else
+          s"""[${info.pos.source}]:${info.pos.line}: ${info.msg}"""
       }
       .mkString("\n")
   }
 
+
+  private def constructParam(name: String, values: Iterable[String]) = {
+    if (values.isEmpty) ""
+    else s"-P:classpath-shrinker:$name:${values.mkString(":")}"
```

Comment:
This also feels too important to be buried.
Maybe the unit test isn't the place for both of them but aren't they an important part of the API?